.github/workflows/sycl-post-commit.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       build_artifact_suffix: sprod_shared
       build_configure_extra_args: --shared-libs --no-assertions --hip --cuda --native_cpu --cmake-opt="-DSYCL_ENABLE_STACK_PRINTING=ON" --cmake-opt="-DSYCL_LIB_WITH_DEBUG_SYMBOL=ON"
       # Docker image has last nightly pre-installed and added to the PATH
-      build_image: "ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:build"
+      build_image: "ghcr.io/intel/llvm/ubuntu2204_build:latest-cc1dd89176dd35522d57022a57204b41ae2611a9"
```

Comment:
Shouldn't this be a `sycl_ubuntu2204_nightly` image instead of a `ubuntu2205_build` image? Maybe like `build-cfd0d41a6a3b2e7a25fdea41202c7ad12091bde2`

From https://ghcr.io/intel/llvm/sycl_ubuntu2204_nightly

Reply:
I'm not sure whether it matters, but I can change it. I thought I'd just use the same image to be consistent with expected results.

Reply:
pre-commit builds with gcc, post-commit builds with clang, looking at the error message, I suspect it's using the nightly's DPC++ build to provide clang++, so I'm not sure it would work with the regular build image

Reply:
@npmiller is correct and I should have noticed it when reviewing :(

Reply:
> pre-commit builds with gcc, post-commit builds with clang, looking at the error message, I suspect it's using the nightly's DPC++ build to provide clang++, so I'm not sure it would work with the regular build image

I see that makes sense now. I've updated it to use the last tagged nightly build, from 14 days ago.